3_Implementation/documentation/html/search/all_9.js

```diff
@@ -0,0 +1,7 @@
+var searchData=
+[
+  ['readme_2emd_22',['Readme.md',['../_readme_8md.html',1,'']]],
+  ['resettest_23',['resetTest',['../unity_8h.html#afb3a9b98e779c4f69e72aca5aa9fa1d7',1,'unity.h']]],
+  ['return_5fif_5ffail_5for_5fignore_24',['RETURN_IF_FAIL_OR_IGNORE',['../unity_8c.html#abf9f99b384602e58874744dda3492aaa',1,'unity.c']]],
+  ['run_5ftest_25',['RUN_TEST',['../unity__internals_8h.html#a27cf08abbe33174b41d808a147cfef11',1,'unity_internals.h']]]
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/2015pushkar/C-mini-Project-303459-LTTS/pullRequest?prid=7744834)
```suggestion
  ["run_5ftest_25",['RUN_TEST',['../unity__internals_8h.html#a27cf08abbe33174b41d808a147cfef11',1,'unity_internals.h']]]
```